about.css

```diff
@@ -0,0 +1,382 @@
+@font-face {
+  font-family: Cocogoose;
+  src: url(./assets/fonts/cocogoose-pro.regular.ttf);
+}
+
+@font-face {
+  font-family: Coconum;
+  src: url(./assets/fonts/Cocogoose\ Pro\ Italic-trial.ttf);
+}
+
+@media screen and (max-width: 768px) {
+  .social-navbar {
+    display: none;
+  }
+
+  .desktop-title {
+    display: none;
+  }
+
+  .navbar-items {
+    display: none;
+  }
+}
+
```

Comment:
Building a mobile first app means writing your media queries from bottom up i.e using `min-width` , everything above the `min-width` will make for a larger screen with specific breakpoints and vice versa.

- [ ] Please use the mobile-first approach only 🙂